test/format/config.c

```diff
@@ -256,24 +256,25 @@ config_compression(const char *conf_name)
 		switch (mmrand(NULL, 1, 20)) {
 		case 1: case 2: case 3: case 4:		/* 20% no compression */
```

Comment:
Would you mind shuffling this around so the no-compression case becomes a default and update the "20% no compression" comment to reflect that the proportion varies depending on what compression options are available?